projects/ag-grid/src/modules/ag-grid/fixtures/ag-grid-data-manager.component.fixture.ts

```diff
@@ -20,7 +20,6 @@ import { SKY_AG_GRID_DATA } from './ag-grid-data.fixture';
 @Component({
   selector: 'sky-ag-grid-data-manager-component-fixture',
   templateUrl: './ag-grid-data-manager.component.fixture.html',
-  styleUrls: ['../../../styles/ag-grid-styles.scss'],
```

Comment:
These imports are causing problems in Nx monorepo. Removing them had no effect on the existing unit tests.